abipy/flowtk/works.py

```diff
@@ -1967,3 +1968,156 @@ def on_all_ok(self):
         # Merge DDB files.
         out_ddb = self.merge_ddb_files()
         return self.Results(node=self, returncode=0, message="DDB merge done")
+
+
+class ConducWork(Work):
+    """
+    Workflow for the computation of electrical conductivity.
+
+    Can be called from :
+        1. MultiDataset and PhononWork
+        2. MultiDataset, DDB filepath and DVDB filepath.
+    Can use Kerange Capability using withKerange=True
+
+    This work consists of 3-4 tasks or 5-6 tasks with kerange :
+        1. SCF GS
+        2. NSCF
+        3. Kerange (Kerange only)
+        4. WFK Interpolation (Kerange only)
+        5. DVDB Interpolation (Optional)
+        6. Electrical Conductivity Calculation.
+    """
+
+    @classmethod
+    def from_phwork(cls, phwork, multi, nbr_proc=None, flow=None, withKerange=False,
```

Comment:
Just a minor comment on the style. We usually try to follow PEP8 for naming conventions although I don't enforce these results via scripts.
We usually use CamelCase for classes while arguments are in snake case (words separated by underscores).